build.gradle

```diff
@@ -12,7 +12,32 @@ subprojects {
     group = "com.netflix.${githubProjectName}" // TEMPLATE: Set to organization of project
 
     repositories {
-        jcenter()
+        File settingsFile = new File(m2SettingsFile.replace('~', System.env.HOME.toString()))
+        if (settingsFile.isFile()) {
+            println "Detected m2 settings file: ${settingsFile}"
```

Comment:
We really need to be using https://docs.gradle.org/current/userguide/dependency_management.html#sec:repositories if at all possible.